Fixtures/BinaryTargets/Framework.xcframework/macos-x86_64/Framework.framework/Versions/A/Modules/Framework.swiftmodule/x86_64.swiftinterface

```diff
@@ -0,0 +1,9 @@
+// swift-interface-format-version: 1.0
```

Comment:
We probably don't need the above swiftdoc binary file.

Reply:
I just generated an XCFramework using Xcode and the standard procedure. Do you want me to remove just this file?

Reply:
Yeah. I don't think we really need it.

Reply:
Done in next commit.